gtk/src/ghb-dvd.c

```diff
@@ -127,14 +127,14 @@ get_buffer(int fd, guint64 off, gsize len)
 }
 
 static gint
-set_unicode16(guint8 *str, gsize len, const guint8 *buf, gint endianess, gsize count)
+set_unicode16(guint8 *str, gsize len, const guint8 *buf, gint endianness, gsize count)
```

Comment:
Both spellings were used in this repo, but this spelling is the accepted spelling.